src/component/ResponsiveContainer.tsx

```diff
@@ -90,7 +90,7 @@ export const ResponsiveContainer = forwardRef(
           return currentSizes;
         });
       }
-    }, [getContainerSize]);
+    }, [getContainerSize, onResize]);
```

Comment:
Nice, this might as well be an unrelated bug fix! ❤️

Reply:
Could you please do this change, and the addition of `"plugin:react-hooks/recommended",` as a separate PR? If issues ever come up, we might revert the changes to the Text, and i would love for these two changes to remain independent.